internal/traceprof/traceproftest/baseline.txt

```diff
@@ -0,0 +1,51 @@
+goos: linux
+goarch: amd64
+pkg: gopkg.in/DataDog/dd-trace-go.v1/internal/traceprof/traceproftest
+cpu: Intel(R) Xeon(R) Platinum 8275CL CPU @ 3.00GHz
+BenchmarkEndpointsAndHotspots/direct/hello-world-16             11256594              6497 ns/op             11018 cpu-ns/op             0.01159 pprof-B/op              0.0007238 pprof-samples/op
```

Comment:
I'm rerunning the benchmarks right now and it looks like the recent changes have improved baseline performance (seeing `6038 ns/op`) 🥳 . Will update the full results here when they are ready.